### ISSUE TITLE
[Bug]: M1 mac brew install timescaled failed!

### ISSUE THREAD

**Issue by johnsxiong** [timescale/timescaledb#4202]
### What type of bug is this?

Configuration

### What subsystems and features are affected?

Other

### What happened?

```
(base) ➜  ~ brew install timescaledb
==> Downloading https://ghcr.io/v2/homebrew/core/postgresql/manifests/14.2_1
######################################################################## 100.0%
==> Downloading https://ghcr.io/v2/homebrew/core/postgresql/blobs/sha256:2bdd0a5c963159499d9033afbd67872799a1dd73d9aa0656fc1276d6b
==> Downloading from https://pkg-containers.githubusercontent.com/ghcr1/blobs/sha256:2bdd0a5c963159499d9033afbd67872799a1dd73d9aa0
######################################################################## 100.0%
==> Downloading https://github.com/timescale/timescaledb-tune/releases/download/v0.12.0/timescaledb-tools-0.12.0-darwin.tgz
Already downloaded: /Users/johnxiong/Library/Caches/Homebrew/downloads/c9d1b4a0b26136d8c0bf1a788a2dc44813dee4367380383b39c6d0dca738a566--timescaledb-tools-0.12.0-darwin.tgz
==> Downloading https://github.com/timescale/timescaledb/archive/refs/tags/2.6.0.tar.gz
Already downloaded: /Users/johnxiong/Library/Caches/Homebrew/downloads/93491e206b9406190c61efcf552677f0d0b92ced0791e3636bf25e6a061f75de--timescaledb-2.6.0.tar.gz
==> Installing timescaledb from timescale/tap
==> Installing dependencies for timescale/tap/timescaledb: postgresql and timescaledb-tools
==> Installing timescale/tap/timescaledb dependency: postgresql
==> Pouring postgresql--14.2_1.arm64_monterey.bottle.tar.gz
🍺  /opt/homebrew/Cellar/postgresql/14.2_1: 3,305 files, 44.9MB
==> Installing timescale/tap/timescaledb dependency: timescaledb-tools
🍺  /opt/homebrew/Cellar/timescaledb-tools/0.12.0: 6 files, 27MB, built in 1 second
==> Installing timescale/tap/timescaledb
==> ./bootstrap -DCMAKE_BUILD_TYPE=RelWithDebInfo -DREGRESS_CHECKS=OFF -DTAP_CHECKS=OFF -DWARNINGS_AS_ERRORS=OFF -DLINTER=OFF -DPR
==> make -C build
==> make -C build install DESTDIR=/private/tmp/timescaledb-20220330-40409-17oh487/timescaledb-2.6.0/stage
find: /usr/local/var/postgres*: No such file or directory
find: /usr/local/var/postgres*: No such file or directory
==> Caveats
RECOMMENDED: Run 'timescaledb-tune' to update your config settings for TimescaleDB.

  timescaledb-tune --quiet --yes

IF NOT, you'll need to make sure to update
to include the extension:

  shared_preload_libraries = 'timescaledb'

To finish the installation, you will need to run:

  timescaledb_move.sh

If PostgreSQL is installed via Homebrew, restart it:

  brew services restart postgresql

==> Summary
🍺  /opt/homebrew/Cellar/timescaledb/2.6.0: 38 files, 5.7MB, built in 9 seconds
==> Running `brew cleanup timescaledb`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
==> Caveats
==> timescaledb
find: /usr/local/var/postgres*: No such file or directory
RECOMMENDED: Run 'timescaledb-tune' to update your config settings for TimescaleDB.

  timescaledb-tune --quiet --yes

IF NOT, you'll need to make sure to update
to include the extension:

  shared_preload_libraries = 'timescaledb'

To finish the installation, you will need to run:

  timescaledb_move.sh

If PostgreSQL is installed via Homebrew, restart it:

  brew services restart postgresql

(base) ➜  ~ timescaledb-tune --quiet --yes
exit: could not find postgresql.conf at any of these locations:
/usr/local/var/postgres/postgresql.conf
(base) ➜  ~
```

### TimescaleDB version affected

2.6.0

### PostgreSQL version used

14.2

### What operating system did you use?

Mac OS Monterey 12.3 ARM

### What installation method did you use?

Homebrew

### What platform did you run on?

On prem/Self-hosted

### Relevant log output and stack trace

```bash
find: /usr/local/var/postgres*: No such file or directory
find: /usr/local/var/postgres*: No such file or directory

exit: could not find postgresql.conf at any of these locations:
/usr/local/var/postgres/postgresql.conf
```

**Comment by horzsolt** [comment]
Hi @johnsxiong thanks for submitting this issue. On M1 homebrew uses /opt/homebrew/ instead of /usr/local/, so the macOS installer needs updates. For the time being, can you please check if this workaround works for you:
`timescaledb-tune --quiet --yes --conf-path /opt/homebrew/var/postgres/postgresql.conf`

**Comment by johnsxiong** [comment]
> 

This worked for me , appreciate !!

**Comment by horzsolt** [comment]
hmm, seems I deleted my comment somehow... For the record, it was like this:

On M1 homebrew uses /opt/homebrew/ instead of /usr/local/, so the macOS installer needs updates. For the time being, can you please check if this workaround works for you:
`timescaledb-tune --quiet --yes --conf-path /opt/homebrew/var/postgres/postgresql.conf`

**Comment by skeetmtp** [comment]
Issue is still present, workaround is working, but can we reopen the issue as bug is still present ?

**Comment by svenklemm** [comment]
@skeetmtp Can you check which version of timescaledb-tune you have?
```
%  timescaledb-tune -version
timescaledb-tune 0.14.1 (linux amd64)
```

**Comment by skeetmtp** [comment]
```
timescaledb-tune 0.14.1 (darwin amd64)
```

**Comment by svenklemm** [comment]
Could you post the output of just running `timescaledb-tune` 
```
>  timescaledb-tune
exit: could not find postgresql.conf at any of these locations:
/etc/postgresql/14/main/postgresql.conf
/var/lib/pgsql/14/data/postgresql.conf
/var/lib/postgres/data/postgresql.conf
/var/lib/postgresql/data/postgresql.conf
```

**Comment by svenklemm** [comment]
I just tagged and released 0.14.2 of timescaledb-tune could you test if everything works with that version?
